docs/release_notes/ibexa_dxp_v4.4.md

```diff
@@ -0,0 +1,120 @@
+---
+description: Ibexa DXP v4.4 adds the improvements to the Welcome Page, PIM and Fastly.
+---
+
+# Ibexa DXP v4.4
+
+**Version number**: v4.4
+
+**Release date**: January 27, 2023
+
+**Release type**: [Fast Track](https://support.ibexa.co/Public/service-life)
+
+## Notable changes
+
+### New welcome page
+
+You will be welcomed to the Ibexa Digital Experience Platform by a newly designed welcome page.
+
+![New Welcome Page](4.4_welcome_page.png)
+
+### All-new Ibexa Commerce packages [[% include 'snippets/commerce_badge.md' %]]
+
+This release deprecates all Commerce packages that you've known from previous releases 
+and brings a completely redesigned and reconstructed Commerce offering.
```

Comment:
I will be careful with this "completely" because the Commerce offering is not fully revamped yet :) We continue in the next releases.